TESTS/spm/client_tests/client_ipc_tests.cpp

```diff
@@ -22,7 +22,7 @@
 using namespace utest::v1;
 
 #define MINOR_VER               0
-#define CLIENT_RSP_BUF_SIZE     MBED_CONF_SPM_CLIENT_DATA_TX_BUF_SIZE_LIMIT
+#define CLIENT_RSP_BUF_SIZE     128
```

Comment:
Was there a reason that this (and the other instance) was changed from a constant to a number?

Reply:
we removed the constant MBED_CONF_SPM_CLIENT_DATA_TX_BUF_SIZE_LIMIT from the configuration due to an API change.
this is part of the alignment with the PSA-IPC spec ARMmbed/PSA-IPC-doc#110

Reply:
The spec change allowed us not to copy input buffers. As the read API was modified in a way that prevents TOCTOU attacks.
We no longer pre-allocate buffers for payloads and  removed the configuration from module.
This is the reason for using a hardcoded value in the test.